src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Colors {
+    Red,
+    Blue,
+    Grey,
+    White,
+    Green,
+    Brown,
+    Black,
+    Orange,
+    Yellow,
+    Purple
```

Comment:
```suggestion
    RED,
    BLUE,
    ...
```

элементы списка принято писать большими буквами